experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1538,9 +1553,11 @@ def validate(self, data):
             raise serializers.ValidationError(
                 {"channel": "Channel is required for this application."}
             )
+
         data = super().validate(data)
-        data = self._validate_feature_config(data)
         data = self._validate_feature_configs(data)
+
+        data = self._validate_feature_config(data)
```

Comment:
Yeah I should revert that, it was significant so I was playing around with it but since I added the checks to only run the single feature validation cases for feature configs <= 1 and multifeature validation case against feature configs > 1 this ordering becomes irrelevant.  I'll revert it.